src/components/Figure/styles.module.scss

```
@@ -13,10 +14,22 @@
         line-height: 1em;
         margin-top: 0.5em;
         margin-bottom: 0.5em;
+
+        :global &.inline {
+            position: absolute;
+            bottom: 0;
+            right: 0;
+            padding: 0.3em;
+            opacity: 50%;
+
+            &:hover {
+                opacity: 100%;
+            }
+        }
     }
 }
 
-:globale .card__image {
+:global .card__image {
```

there are some places where `:global <selector>` is used - these were introduced when the propper syntax was not clear to the author ;) It should be

```suggestion
:global(.card__image) {
```

---

That syntax has now also become clear to this author, thanks ;)